phonegap-nfc/phonegap-nfc.d.ts

```diff
@@ -0,0 +1,456 @@
+// Type definitions for Phonegap NFC Plugin
+// Project: https://github.com/chariotsolutions/phonegap-nfc
+// Definitions by: Michael Desigaud <https://github.com/michaeldesigaud>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface Document {
+    addEventListener(type: 'deviceready', listener: (ev: Event) => any, useCapture?: boolean): void;
+}
+
+/**
+ * Global object NFC.
+ */
+interface Window {
+    nfc: Nfc;
+    ndef: Ndef;
+    util:Util;
+    fireNfcTagEvent(event:TagEvent, tagAsJson:string):void;
+}
+
+interface Tag {
```

Comment:
you should use the this pattern.
http://definitelytyped.org/guides/best-practices.html#ghost-modules